Content.Server/Speech/EntitySystems/StutteringSystem.cs

```diff
@@ -37,15 +33,16 @@ private void OnAccent(EntityUid uid, StutteringAccentComponent component, Accent
         public string Accentuate(string message, StutteringAccentComponent component)
         {
             var length = message.Length;
-
             var finalMessage = new StringBuilder();
-
             string newLetter;
 
+            // get regex accent according to stutter.ftl
+            Regex stutter = new(Loc.GetString("stutter-speech-regex-accent"), RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
this will recompile the regex for every message

Reply:
Yeah fixed